README.md

```diff
@@ -0,0 +1,121 @@
+
+    sudo aptitude install graphviz
+
+
+Usage
+-----
+
+In the Markdown source, the Graphviz code must be inserted as an individual block (i.e., separated from the rest of the material by blank lines), like the following:
+
+```markdwon
+..graphviz dot
+digraph G {
+  graph [rankdir = LR];
+  Hello -> World
+}
+```
+
+This will insert an image in your article like this:
+
+![figure](hello-world.png)
+
+The block must start with `..graphviz` (this is configurable — see below). The word `dot` in the first line indicates the program that will be run to produce the image. The available programs are: `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, and `patchwork` (see the [Graphviz documentation][] for details). The Graphviz code must start in the second line of the block. Notice that *newlines are not allowed inside the Graphviz block*.
+
+[Graphviz documentation]: https://www.graphviz.org/documentation/
+
+
+Styling with CSS
+----------------
+
+The image is generated in HTML with an `<img>` element inside an HTML element (by default a `<div>`, but this is configurable — see below). The latter has class `graphviz` (this is also configurable — see below). A possible CSS styling would be:
+
+```css
+div.graphviz  {
+    margin: 10px;
+}
+div.graphviz img {
+    display: block;
+    padding: 5px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    border-style: solid;
+    border-width: 1px;
+    border-color: rgb(192, 177, 177);
+    -webkit-border-radius: 5px;
+}
+```
+
+
+Configuration
+-------------
+
+The following variables can be set in the Pelican settings file:
+
+- `GRAPHVIZ_HTML_ELEMENT`: The HTML inside which the generated Graphviz image is inserted (defaults to `'div'`; another sensible possibility would be `'span'`).
+
+- `GRAPHVIZ_IMAGE_CLASS`: Class of the `<div>` element including the generated Graphviz image (defaults to `'graphviz'`).
+
+- `GRAPHVIZ_BLOCK_START`: Starting tag for the Graphviz block in Markdown (defaults to `'..graphviz'`).
+
+
+Output Image Format
+-------------------
+
+The format of the embedded image is SVG, and there is currently no way to change it. This format was chosen over others (like PNG) for two reasons. First, the generated SRC string in Base64 seem to be shorter for SVG than for PNG. Second, the image will be available in the browser in a high-quality vectorized format. As a caveat, notice that this choice may prevent display in browsers lacking proper SVG support.
+
+
+Alternatives
+------------
+
+An alternative to this plugin would be the Graphviz tag provided by the [Liquid Tags extension][] available at Github. However, this extension needs a patch to the [liquid_tags][] plugin, which complicates its installation.
+
+[Liquid Tags extension]: http://blog.dornea.nu/2014/11/13/using-graphviz-with-pelican-and-liquid-tags/
+[liquid_tags]: https://github.com/dorneanu/pelican-plugins/tree/master/liquid_tags
```

Comment:
Is there a reason you decided to use these older links instead of the new Liquid Tags repo, and its corresponding Graphviz tag?
* https://github.com/pelican-plugins/liquid-tags
* https://github.com/pelican-plugins/liquid-tags/blob/main/pelican/plugins/liquid_tags/graphviz.py

Reply:
No special reason. Please, update the links as it seems better to you.

Reply:
> No special reason. Please, update the links as it seems better to you.

Ok, I just pushed an updated "Alternatives" section (migrate branch) with a detailed explanation of the differences between the Graphviz plugin and the graphviz tag of Liquid Tags.